.github/workflows/ci.yml

```diff
@@ -264,6 +264,26 @@ jobs:
         rustup target add thumbv6m-none-eabi
         cargo check -p cordyceps --target thumbv6m-none-eabi
 
+  # run no-atomics target tests
+  maitake_sync_no_atomics_compat:
+    needs: changed_paths
+    if: needs.changed_paths.outputs.should_skip != 'true' || !fromJSON(needs.changed_paths.outputs.paths_result).maitake.should_skip
+    runs-on: ubuntu-latest
+    name: no-atomics compat (maitake-sync)
+    steps:
+    - name: install rust toolchain
+      run: rustup show
+    - uses: actions/checkout@v4
+    - name: run check
+      # we add the target HERE instead of in the 'install rust toolchain' step
+      # because we need the `rust-toolchain.toml` file to be checked out, otherwise
+      # the target is only added for `stable`. We could also add this to the
+      # `targets` item in the toolchain file, but that would be a waste for all
+      # other tests.
+      run: |
+        rustup target add thumbv6m-none-eabi
+        cargo check -p maitake-sync --target thumbv6m-none-eabi --no-default-features --features=critical-section
+
   ### maitake ###
```

Comment:
can we move this line to above the newly-added CI job? i wanted to include a little section header for each crate that gets tested, and now the new Maitake check isn't under that heading :P